resources/i18n/pl_PL/uranium.po

```diff
@@ -324,11 +326,14 @@ msgid ""
 "{error}.\n"
 "Please try to upgrade again later."
 msgstr ""
+"Wystąpił błąd podczas odinstalowywania pakietu {package} przed instalacją nowej wersji:\n"
+"{error}.\n"
+"Spróbuj zaktualizować ponownie później."
```

Comment:
Last sentence: "Please" is missing.
Try to make the sentence more polite.

Reply:
@drzejkopf is that "Proszę, spróbuj zaktualizować ponownie później." correct for this?

I can do it directly in 3.5.

Reply:
@diegopradogesto I have Polish roots, I can't write that stuff but read and talk :sweat_smile: 
Yes, your text is in my point of view correct. Just don't know whether they place a comma after the "Please". But I guess, it does not hurt to place more commas as in English.

But, yeah, wait for @drzejkopf better.

Reply:
It should be "Proszę spróbować zaktualizować ponownie później." to keep impersonal construction of sentence (it's good practice to use impersonal forms when translating promps to Polish), but it is common that promps and messages like this one are being translated without polite forms. Both versions are correct, none of them is impolite, but if you want more straightforward translation, you can use form I mentioned above :)

Reply:
Ok. Thanks to both of you. Done in 9c04fcf89516dc3972efbe0d9df5bc71d561be4b